src/app/app.component.html

```diff
@@ -1,60 +1,87 @@
-<div class="container">
-  <div class="row">
-    <div class="col-md-4 centered" *ngFor="let product of products">
-      <div *ngIf="product.type == 'dish'">
-        <h1>{{product.name}} {{product.price}}:-</h1>
-        <img class="product_image" data-toggle="modal" attr.data-target="#productModal{{product.id}}"
-             src={{product.image}}/>
-
+<div class="body">
+  <nav class="navbar navbar-default">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <img class="logo" src="../assets/kimchistan.jpg">
+      </div>
+    </div>
+    <div class="header-text">
+      <h5> Open: Monday - Friday 11-19</h5>
+      <h5> Order between: 11-18:30</h5>
+    </div>
+  </nav>
+  <br>
+  <br>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-md-3">
+        <img class="sidebar_image"
```

Comment:
I think we should rename it to sidedish_image or something like this. It's not a sidebar, and could confuse others. :)

Reply:
Are we not displaying opening hours during weekends? 
and linking to the about us page?

Reply:
This story is about logo and background. I'll fix open during weekend. @olofvi We can have a link when we create about us page. :)
`sidedish_image` 👍  @Sajaas

Reply:
just confusing because we were working on this story earlier https://www.pivotaltracker.com/story/show/152638119